discovery-infra/test_infra/helper_classes/kube_helpers/cluster_deployment.py

```diff
@@ -50,7 +53,7 @@ def as_dict(self) -> dict:
             'agentBareMetal': {
                 'apiVIP': self.api_vip,
                 'ingressVIP': self.ingress_vip,
-                'agentSelector': self.agent_selector or {}
+                'agentSelector': self.agent_selector or {},
```

Comment:
was this comma added intentionally?

Reply:
Yes, to make diffs smaller if someone adds items
If you hate it I can remove all of those